supportedBackends/htcondor/src/main/scala/cromwell/htcondorbackend/HtCondorBackend.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.htcondorbackend
```

Comment:
These package names are all two words smashed together, can they be broken out into two components?